validator/src/test/java/com/amazon/aoc/validators/CWLogValidatorTest.java

```diff
@@ -0,0 +1,151 @@
+package com.amazon.aoc.validators;
+
+import static org.junit.Assert.*;
+import static org.mockito.ArgumentMatchers.*;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+import com.amazon.aoc.callers.HttpCaller;
+import com.amazon.aoc.exception.BaseException;
+import com.amazon.aoc.exception.ExceptionCode;
+import com.amazon.aoc.models.CloudWatchContext;
+import com.amazon.aoc.models.Context;
+import com.amazon.aoc.models.SampleAppResponse;
+import com.amazon.aoc.models.ValidationConfig;
+import com.amazon.aoc.services.CloudWatchService;
+import com.amazonaws.services.logs.model.OutputLogEvent;
+import com.github.fge.jsonschema.report.ProcessingReport;
+import java.util.ArrayList;
+import java.util.List;
+import org.junit.Test;
+
+public class CWLogValidatorTest {
+  @Test
+  public void testSuccessfullLogMessage() throws Exception {
+    ValidationConfig validationConfig = new ValidationConfig();
+    validationConfig.setCallingType("http");
+    validationConfig.setExpectedLogStructureTemplate(
+        "file://"
+            + System.getProperty("user.dir")
+            + "/src/main/resources/expected-data-template/otlp/otlp-log.json");
+    Context context = initContext();
+
+    String message =
+        "{\n"
+            + "    \"body\": \"Executing outgoing-http-call\",\n"
+            + "    \"severity_number\": 9,\n"
+            + "    \"severity_text\": \"INFO\",\n"
+            + "    \"flags\": 1,\n"
+            + "    \"trace_id\": \"6541324dc3026f11c99345889da1a47d\",\n"
+            + "    \"span_id\": \"c6f853c5f487c5e6\",\n"
+            + "    \"resource\": {\n"
+            + "        \"service.name\": \"aws-otel-integ-test\"}}";
+
+    OutputLogEvent outputLogEvent = new OutputLogEvent();
+    outputLogEvent.setMessage(message);
+    List<OutputLogEvent> eventList = new ArrayList<>();
+    eventList.add(outputLogEvent);
+    CWLogValidator validator = runValidation(validationConfig, context, eventList);
+    ProcessingReport report = validator.getProcessingReport();
+    assertTrue(report.isSuccess());
```

Comment:
```suggestion
```
These assertions should not be required. No exception being thrown indicates the test case is passing right?